integration-tests/search.test.ts

```diff
@@ -117,6 +117,46 @@ describe('search', () => {
         }
     });
 
+    test('tokens', async () => {
+        const randomPatientData = randomPatient();
+        randomPatientData.identifier = [
+            {
+                system: 'http://fwoa-integ-tests.com',
+                value: 'someCode',
+            },
+        ];
+        const testPatient: ReturnType<typeof randomPatient> = (await client.post('Patient', randomPatientData)).data;
+
+        const randomPatientDataNoSystem = randomPatient();
+        randomPatientData.identifier = [
+            {
+                value: 'someCodeWithoutSystem',
+            },
+        ];
+        const testPatientNoSystem: ReturnType<typeof randomPatient> = (
+            await client.post('Patient', randomPatientDataNoSystem)
+        ).data;
+
+        // wait for the patient to be asynchronously written to ES
+        await waitForPatientToBeSearchable(client, testPatient);
+        await waitForPatientToBeSearchable(client, testPatientNoSystem);
+
+        const aFewMinutesAgo = new Date(Date.now() - 1000 * 60 * 10).toJSON();
+        const p = (params: any) => ({ url: 'Patient', params: { _lastUpdated: `ge${aFewMinutesAgo}`, ...params } });
+
+        const testsParamsThatMatch = [
+            p({ identifier: 'http://fwoa-integ-tests.com|someCode' }),
+            p({ identifier: 'someCode' }),
+            p({ identifier: 'http://fwoa-integ-tests.com|' }),
```

Comment:
Would the empty ('') search work here too?

Reply:
It is a valid query, so you get a 200 response with an empty bundle.  But it would fail the test since it doesn't match any patients

Reply:
got it thanks